### PR TITLE
Part of #18714: Replacing deprecated constants

### PR DESCRIPTION
**Pull request by dhruvv173**
## Explanation

- This PR is a part of #18714 
- #19526 

Made changes to the following files

1. `ui/components/app/qr-hardware-popover/qr-hardware-sign-request/player.js`
2. `ui/components/app/signature-request-siwe/signature-request-siwe-icon/index.js`
3. `ui/components/app/signature-request-siwe/signature-request-siwe-tag/index.js`
4. `ui/components/app/tab-bar/tab-bar.js`

### Created story for `Player`
image
image

### Before
image
image
### After
image
image

### Before
image
image

### After
image
image

### Before
image
image

### After
image
image

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/components/app/qr-hardware-popover/qr-hardware-sign-request/player.js
Comment (dhruvv173) [inline ui/components/app/qr-hardware-popover/qr-hardware-sign-request/player.js:1]: hey @georgewrmarshall, just a quick question before you review this PR. This file doesnt have a storybook file, would you mind if I create one so its easier for the maintainers to review it?
Thanks!

Comment (georgewrmarshall) [reply ui/components/app/qr-hardware-popover/qr-hardware-sign-request/player.js:1]: That would be great@

Comment (dhruvv173) [reply ui/components/app/qr-hardware-popover/qr-hardware-sign-request/player.js:1]: Done. Please check when possible, thanks!

**Review by georgewrmarshall — APPROVED** [review]
LGTM! Thanks for your contribution

**Review by NidhiKJha — APPROVED** [review]

**Pull request merged**